api/WindowOrWorkerGlobalScope.json

```
@@ -1010,6 +1010,51 @@
           }
         }
       },
+      "queueMicrotask": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/queueMicrotask",
+          "support": {
+            "chrome": {
+              "version_added": "71"
+            },
+            "chrome_android": {
+              "version_added": "71"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "69"
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "58"
+            },
+            "opera_android": {
+              "version_added": false
```

Is this actually false? Seems kinda odd, I would've guessed 50, unless Opera went out of their way to not ship it on Android.

```suggestion
              "version_added": "50"
```

---

I just left it at `false` because it errored out and I wasn't sure what version to put. 50 is fine by me if it's fine by you!